index.js

```diff
@@ -0,0 +1,12 @@
+const hamburger = document.querySelector('.hamburger');
+const navMenu = document.querySelector('.nav-menu');
+
+hamburger.addEventListener('click', () => {
+  hamburger.classList.toggle('active');
+  navMenu.classList.toggle('active');
+});
+
+document.querySelectorAll('.nav-link').forEach((n) => n.addEventListener('click', () => {
+  hamburger.classList.remove('active');
+  navMenu.classList.remove('active');
+}));
```

Comment:
- [ ] Excellent work :clap: , however please notice that we have to setup EsLint for this project, since you have started using Javascript, it is now required that you setup eslint also along with the HTML and CSS linters, so [copy the eslint.json file from here](https://github.com/microverseinc/linters-config/blob/master/html-css-js/.eslintrc.json) and also update the[ linter.yml file from here](https://github.com/microverseinc/linters-config/blob/master/html-css-js/.github/workflows/linters.yml) and make sure your eslint checks are running and passing :+1: